static/.htaccess

```diff
@@ -3,6 +3,17 @@ RewriteEngine on
 RewriteCond %{SERVER_PORT} !^443$
 RewriteRule (.*) https://%{SERVER_NAME}/$1 [R=301,L]
 
+# Rename Console API
+RewriteRule ^docs/apis-clients/console-api/?$ /docs/apis-clients/administration-api/$1 [R=301,L]
+RewriteRule ^docs/next/apis-clients/console-api/?$ /docs/next/apis-clients/administration-api/$1 [R=301,L]
+RewriteRule ^docs/8.4/apis-clients/console-api/?$ /docs/8.4/apis-clients/administration-api/$1 [R=301,L]
+RewriteRule ^docs/apis-clients/console-api/console-api-reference/?$ /docs/apis-clients/administration-api/administration-api-reference/$1 [R=301,L]
+RewriteRule ^docs/next/apis-clients/console-api/console-api-reference/?$ /docs/next/apis-clients/administration-api/administration-api-reference/$1 [R=301,L]
+RewriteRule ^docs/8.4/apis-clients/console-api/console-api-reference/?$ /docs/8.4/apis-clients/administration-api/administration-api-reference/$1 [R=301,L]
+RewriteRule ^docs/8.3/apis-clients/console-api-reference/?$ /docs/8.3/apis-clients/administration-api-reference/$1 [R=301,L]
+RewriteRule ^docs/8.2/apis-clients/console-api-reference/?$ /docs/8.2/apis-clients/administration-api-reference/$1 [R=301,L]
+RewriteRule ^docs/8.1/apis-clients/console-api-reference/?$ /docs/8.1/apis-clients/administration-api-reference/$1 [R=301,L]
+
```

Comment:
@christinaausley I think you intended for these rewrites to specify `apis-tools` instead of `apis-clients` (on both the left and right sides)!

Reply:
![facepalm-really](https://github.com/camunda/camunda-docs/assets/84338309/0da44822-ef8d-415b-b28e-3a94e1b08be1)

Reply:
Maybe you were temporarily possessed by Christina from 1 year ago, pre-rename 😅